src/CopyrighFixer/CopyrighFixer/signer.cpp

```diff
@@ -18,6 +19,15 @@ bool Signer::checkSign(const Config &objConf) {
 
 IFileManager *Signer::searchFileByExt(const QString &extension) {
 
+    for (auto itemFM: FileManager) {
+
+        if (itemFM && itemFM->isSupport(itemFM->toExtension(extension))) {
+            return itemFM;
+        }
+
+        return nullptr;
+    }
+
```

Comment:
i thin you mean this :
```suggestion

    }
    return nullptr;
```